src/models/TwitterInfo.ts

```diff
@@ -8,54 +8,59 @@ export default class TwitterInfo {
   /**
```

Comment:
If it implements the interface, add the `implements` bit to the class declaration on line 7 (GH won't let me add a suggestions)